pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -14,8 +14,8 @@ import (
 	"github.com/samalba/dockerclient"
```

Comment:
Agree so I think it is best (to worst):
1 - First Do the engine-api refactor and avoid introducing new code which still uses the old client, only then add the volumes code.
2 - remove the unused import from the imports and continue using the old client
3 - Do nothing at all